src/elements/content-preview/ContentPreview.js

```diff
@@ -1081,6 +1083,12 @@ class ContentPreview extends PureComponent<Props, State> {
         this.previewContainer = container;
     };
 
+    getSidebarRef = (): ?React.Node => this.sidebarRef;
```

Comment:
Can we make it so that there's a `contentSidebarRef` prop that we forward to `ContentSidebar`? That way we don't have to worry about setting it up ourselves.